stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1618,7 +1618,7 @@ public class ChatClient internal constructor(
         private var callbackExecutor: Executor? = null
         private var loggerHandler: ChatLoggerHandler? = null
         private var notificationsHandler: NotificationHandler? = null
-        private var notificationConfig: NotificationConfig = NotificationConfig()
+        private var notificationConfig: NotificationConfig = NotificationConfig(pushNotificationsEnabled = false)
```

Comment:
Is it a breaking change? Maybe we should mention it in the changelog?

Reply:
it's not a breaking change because you won't receive a push notification until you pass `PushDeviceGenerator` and `pushNotificationsEnabled` is set to `true` by default when you are creating a new `NotificationConfig`